cypress/support/step_definitions/validations.js

```diff
@@ -49,10 +49,13 @@ Given(
         fireboltItems.forEach((item) => {
           const validationType = item.event ? CONSTANTS.EVENT : CONSTANTS.METHOD;
 
-          const methodOrEvent = item[validationType];
+          let methodOrEvent = item[validationType];
+          methodOrEvent = methodOrEvent.includes('_') ? methodOrEvent.split('_')[1] : methodOrEvent;
           const context = item.context ? item.context : CONSTANTS.NO_CONTEXT;
-          const validationJsonPath = item.validationJsonPath;
-          const expected = item.expected;
+          const validationJsonPath = item.validationJsonPath
```

Comment:
Can we put a quick comment here to explain why we're setting "result" and "null" as defaults?

Reply:
added comment as :
          // if the validationObject doesn't have validationJsonPath or expected fields, assign default values.